__tests__/app.test.js

```diff
@@ -8,6 +8,20 @@ const app = require("../app");
 beforeEach(() => seed(testData));
 afterAll(() => connection.end());
 
+describe("/api", () => {
+  describe("GET", () => {
+    test("200 OK: should return all of the endpoints in endpoints.json", () => {
+      return request(app)
+        .get("/api")
+        .expect(200)
+        .then((endpoints) => {
+          Object.keys(endpoints).forEach((endpoint) => {
```

Comment:
This is unnecessary and we can also be more specific with what we're expecting back, by simply requiring the endpoints json into your test file and asserting that the response matches it in it's entirety.